docs/content/users/basics/developer-tools.md

```diff
@@ -86,34 +88,35 @@ MAIL_ENCRYPTION=null
+If you need to connect to your project’s database from your host machine, run `ddev describe` to show the database connection information, like `Host: localhost:49156`.
 
-The port referenced is unique per running project, and randomly chosen from available ports on your system when you run `ddev start`.
+Each project’s database port is unique, and randomly chosen from available ports on your system when you run `ddev start`.
 
-You can force this port to be the same on every `ddev start` by setting `host_db_port` in the project .ddev/config.yaml. For example, `host_db_port: "49156"` or `ddev config --host-db-port=49156`. This value needs to be different on each running DDEV project, and unless it is set, the database port will change on every `ddev start`.
+You can force this port to be the same on every `ddev start` by setting `host_db_port` in the project `.ddev/config.yaml`. For example, `host_db_port: "49156"` or `ddev config --host-db-port=49156`. This value needs to be different on each running DDEV project, and unless it is set, the database port will change on every `ddev start`.
 
-You can use this port with various tools that need a direct port, like `mysql` or `psql` clients, but it's usually easiest to use `ddev mysql`, `ddev psql`, `ddev sequelace`, `ddev tableplus`, etc, which set everything up for you.
+You can use this port with various tools that need a direct port, like `mysql` or `psql` clients, but it’s usually easiest to use `ddev mysql`, `ddev psql`, `ddev sequelace`, `ddev tableplus`, etc, which set everything up for you.
 
-## Using Drush 8 installed Installation on the Host Computer
+## Using Drush 8 Installation on Your Host Machine
 
-**Warning:** Using drush on the host is discouraged, and you'll have some trouble with it. It's also mostly irrelevant for Drupal8, as you should be using composer-installed project-level drush.
+!!!warning
+    We don’t recommend using `drush` on your host machine. It’s also mostly irrelevant for Drupal 8, as you should be using Composer-installed, project-level `drush`.
 
-If you have PHP and Drush installed on your host system and the environment variable IS_DDEV_PROJECT=true, you can use drush to interact with a ddev project. On the host system the extra include host-side configuration for the database and port are derived in the settings.ddev.php file to allow drush to access the database server. This may not work for all drush commands because of course the actual webserver environment is not available.
+If you have PHP and Drush installed on your host system and the environment variable `IS_DDEV_PROJECT=true`, you can use Drush to interact with a DDEV project. On the host machine, extra host-side configuration for the database and port in `settings.ddev.php` allow Drush to access the database server. This may not work for all Drush commands because the actual web server environment is not available.
```

Comment:
The GitHub repo you just linked to capitalizes it at the end of a sentence, and [drush.org](https://www.drush.org/latest/) does the same thing all over the place. They seem particularly careful about treating it as a proper noun, or otherwise using quotes or italics.

![Screen Shot 2022-09-26 at 10 47 54 AM@2x](https://user-images.githubusercontent.com/2488775/192345591-e2ad52de-b9af-42a5-8e33-36b9df68ec24.png)

![Screen Shot 2022-09-26 at 10 49 04 AM@2x](https://user-images.githubusercontent.com/2488775/192345859-4a6996c1-b5fb-44b8-8044-1a116754c020.png)

Reply:
A-OK. I don't think drush.org has any official connection with the project, but "Drush" is fine!

Reply:
🤷‍♂️ I honestly don’t know much about Drush, they just linked drush.org prominently from [that GitHub repo](https://github.com/drush-ops/drush)—both in repository settings and the readme.